src/v2/coins/abstractUtxoCoin.ts

```diff
@@ -13,6 +13,32 @@ const errors = require('../../errors');
 const debug = require('debug')('bitgo:v2:utxo');
 
 class AbstractUtxoCoin extends BaseCoin {
+
+  public static readonly AddressTypes = Object.freeze({
+    P2SH: 'p2sh',
+    P2SH_P2WSH: 'p2sh-p2wsh',
+    P2WSH: 'p2wsh',
+  });
+
+  public static readonly AddressTypeChains = Object.freeze({
```

Comment:
@paddyquinn is currently working to integrate use of https://github.com/BitGo/unspents across the SDK for transaction size estimation. We should also use it for the chain codes across the board.

Reply:
FYI typescript has enums that can evaluate here above https://www.typescriptlang.org/docs/handbook/enums.html

Reply:
I think we'll integrate unspents lib in a subsequent change (that feels out of scope right now), but I am aware of its existence.

Reply:
Replacing this with the unspents lib is already on my list @Xekyo :) Definitely out of scope for this PR. I will take care of it in a separate PR.

Reply:
Yeah, I meant it as an FYI to help Tyler forgo doing the work here.